.gitignore

```diff
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+package-lock.json
```

Comment:
Why is this added to git ignore? It's a crucial file to every project, read more [here](https://docs.npmjs.com/files/package-lock.json).